admin-dev/themes/default/.eslintrc.js

```diff
@@ -18,15 +18,42 @@ module.exports = {
     parser: 'babel-eslint',
   },
   extends: [
-    'prestashop',
+    'airbnb-base',
```

Comment:
Why this change ?

Reply:
This comes from a public discussion with @SharakPL 

It's a pain in the ass to manage dependency upgrades with a common configuration package, for like 8 rules, it's really not worth

Reply:
:+1: